Modules/_io/textio.c

```diff
@@ -2395,8 +2392,8 @@ _io_TextIOWrapper_tell_impl(textio *self)
         } \
```

Comment:
Okay, this can be done in a separate PR. But in any case I think it is worth to add a NEWS.d entry.